### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by aldemirenes**
Instructions for integrating ScalaDoc-GhPages can be found in [here](https://gist.github.com/aldemirenes/3a41b2e808e82053b3d20a06e8a17fc3).

### CODE REVIEW

**Comment by aldemirenes** [comment]
I've realized some problem with scoverage. It causes `NoDefFoundError` while trying to use library. I've made some research but couldn't find a definite answer. Error can be generated with publishing library local with `+publishLocal` command and trying to use it. Some related link can be found in [here](https://blog.ssanj.net/posts/2016-01-14-scoverage-breaks-with-java-lang-NoClassDefFoundError-scoverage-Invoker.html).

File: .travis.yml [hunk 1]
```diff
@@ -23,3 +23,4 @@ env:
   - secure: R3N94P7noue7/K2KOLjfxNVZV6Eft2emrU0zfh/qbkAwW6CF6w3gjVxRJwBzOhrsy5W4SU7r18Dft9/YS1QQOKw5HuBGNKnVytJxhkHPRB7EE/KrDnlQ0uHK9LAlQnwQzb74+iyQv6ONcrCF1M7xEnnkst8buZp4zoB8JXUA8bM=
   - secure: jh4RLrx+HoTmzgfvsxle8hWdAsz6m5M0qvaBu3VAGPnrM4NmRPyfyb08btlN+qLJOTEBilZGIKh+cYSE8NRJHXR+8iAKHikop8rOh6HqwaEl8A7SFy/Nx6RMys3Cy3yX5AVLHhgpu4absok4QlVplYhn2aK6EbYsM0VS1uyOztQ=
   - secure: ZdJH16Iz79TTwPCha92BuekZP07QsLAxMqaG4liH45ncK4TE+xfEE0lAmt2SqonPodXDbx94cFwjTEum0xFsEOp1ASsz393i265+qVKT+WwumpMe1/4Ta9UF6bAZMBQSeikgW7Qnh7uN2DVUKJ9LNrCg3xlBReWeqJC/1AjD+TM=
+  - ENCRYPTION_LABEL: <placeholder>
```
Comment (dilyand) [inline .travis.yml]: Shouldn't this be removed if we're not using it?

On the other hand, should we be using it, to make the GH publishing consistent across projects?

Comment (chuwy) [reply .travis.yml]: Yes! You're right

File: src/test/scala/com.snowplowanalytics.iglu.client/validator/RawValidationSpec.scala [hunk 2]
```diff
@@ -111,7 +112,7 @@ class RawValidationSpec extends Specification with DataTables {
     val doubleError = CirceValidator.validate(doubleErrorInput, simpleSchemaResult) must beLeft(
       doubleErrorExpected)
 
-    nonString and missingKey and heterogeneusArray
+    nonString and missingKey and heterogeneusArray and doubleError
```
[the remark this answers is not in the document]
Comment (chuwy) [reply src/test/scala/com.snowplowanalytics.iglu.client/validator/RawValidationSpec.scala:115]: That's some very deep review.

File: .travis.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ deploy:
   provider: script
   script: ./.travis/deploy.sh $TRAVIS_TAG
   on:
-    condition: '"${TRAVIS_SCALA_VERSION}" == "2.12.8" && "${TRAVIS_JDK_VERSION}" == "openjdk8"'
+    condition: '"${TRAVIS_SCALA_VERSION}" == "2.12.8"'
```
[the remark this answers is not in the document]
Comment (chuwy) [reply .travis.yml:18]: TBH, I don't even know why we need several version in the first place. SBT is responsible for cross-version test, not Travis.

**Pull request merged**